happybase/table.py

```diff
@@ -558,6 +559,25 @@ def counter_inc(self, row, column, value=1):
         return self.connection.client.atomicIncrement(
             self.name, row, column, value)
 
+    def counter_incs(self, row, data):
```

Comment:
Perhaps `counters_inc` is a better name? And can you move it below `counter_dec` for clarity?